libdnf/rpm/solv/goal_private.hpp

```diff
@@ -116,6 +118,9 @@ class GoalPrivate {
 
     libdnf::solv::Pool & get_pool() { return libdnf::get_pool(base); };
 
+    /// @return True if SOLVER_CLEANDEPS flag was set for any of jobs
+    bool get_clean_deps_present() { return clean_deps_present; }
```

Comment:
[minor] I know it is difficult to name it and it is not wrong. What about to not use `get` word but something else. Like
`is_clean_deps_present`, `using_clean_deps`, `contain_clean_deps`, ... 
But as I said, it is not a problem.

Reply:
I guess `is_clean_deps_present` will do. I'm grateful for every advice, because coming up with a name is the hardest part :)